### ISSUE TITLE
We need interested people

### ISSUE THREAD

**Issue by TheMrIron2** [Game-Fusion/SMES#2]
We need people interested in developing or buying a successor to the seemingly popular MES. Even though MES seems to be a hit with developers intrigued, the SMES is unknown. Who would be willing to help? I'm giving @Game-Fusion/rare read and write permissions so they can see how the console works and fix issues, and @Game-Fusion/consoles admin [obviously].

**Comment by MCFunRide** [comment]
Ok iron, I'm considering on taking over the project, if that is what you are looking for. I'll revamp it's UI and give it a modern feel.

**Comment by byte-chan** [comment]
> it's UI

its*

**Comment by TheMrIron2** [comment]
No, the purpose of the MES isn't to be a modern console, but to be monochrome and to be different. Adds some flavour to the unvaried and frankly dull competition (most of the competitors' Consoles are alike, like Aurorie's UI being near identical to RedCast and Dawn's UI using the same core UI style as GS2, a console more than half a year older than it). Anyway, glad to have you onboard.

Mult, while you're here, want to help out?

**Comment by byte-chan** [comment]
> want to help out?

maybe
